src/System.Console/tests/SyncTextReader.cs

```diff
@@ -0,0 +1,206 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using Xunit;
+
+public class SyncTextReaderTests
+{
+    // NOTE: These tests test the underlying SyncTextReader by
+    // accessing the Console.In TextReader (which in fact is a SyncTextReader).
+
+    static readonly string[] s_testLines = new string[] {
```

Comment:
I looked at the history but its unclear why we chose this set. I will create an issue to review this. Also, in general for common constants we tend to create a helper class.